docs/howto/openstack/nova/new-server.md

```diff
@@ -30,15 +30,15 @@ file first](../../getting-started/enable-openstack-cli.md).
     new pane titled _Create_ will slide into view from the right-hand side
     of the browser window.
 
-    ![Create new object](assets/new-server/shot-01.png)
+    ![Create new Cleura Cloud object](assets/new-server/shot-01.png)
```

Comment:
Does anything break if you use a macro here?

```suggestion
    ![Create new {{brand}} object](assets/new-server/shot-01.png)
```

Reply:
Nothings breaks :)